File: lib/fizzbuzz.rb
def fizzbuzz num
  if num % 5 == 0 && num % 3 == 0
    'fizzbuzz'
  elsif num %5 == 0 
    'buzz'
  elsif num %3 == 0
    'fizz'
  elsif num.class != Integer || num < 0
    "ERROR Give me an integer from zero to 100"
  else
    num
  end
end

File: src/fizzBuzz.js
var FizzBuzz = function() {
};

FizzBuzz.prototype = {
  play: function(number) {
    if(number % 15 === 0) {
      return this.play = 'FizzBuzz'
    };
    if(number % 5 === 0) {
      return this.play = 'Buzz';
    };
    if(number % 3 === 0) {
      return this.play = 'Fizz';
    };
    return this.play = number;
  }
};
